### DISCUSSION TITLE
Discretization of partial differential equations (PDE) in topological structure

### DISCUSSION THREAD

**Discussion by LS-Maxwell** [PMEAL/OpenPNM#2338] [Q&A] [answered]
Woo...,OpenPNM is a very nice code  and I can easily use it for my pore media  simulation with pore and throat mode simulation.

I want to compute the flow problem with PDE equations in the topology structure by pore and throat model , Vornonoi structure for example.  But,  I can not find  some detail description about how to **discrete the equations in  topology structure** by finite difference method(FDM) or finite volume method(FVM).   But I have to solve partial differential equations in this complex structure.
 So , could you give me some suggestions or about it? 

![p1](https://user-images.githubusercontent.com/94354778/157191566-abbea9e7-ab89-4d14-bf7d-7b7795e27add.png)

**Comment by ma-sadeghi** [comment, marked as answer]
If I understood correctly, you should look into the `algorithms` module. [Here](https://openpnm.org/examples/examples.html#performing-simulations), you can find some examples, which explain how to run different algorithms.
